src/grimp/application/ports/graph.py

```diff
@@ -32,6 +32,16 @@ def modules(self) -> Set[str]:
         """
         raise NotImplementedError
 
+    @abc.abstractmethod
+    def find_matching_modules(self, expression: str) -> Set[str]:
+        """
+        Find all modules matching the passed expression.
+
+        See https://grimp.readthedocs.io/en/stable/usage.html#module-expressions
+        for a description of module expression syntax.
```

Comment:
🍷  Rather than duplicate all the information here about "what are module expressions" (and also in `find_matching_direct_imports`), I thought I'd just refer to the docs. Is that sensible?

Reply:
I've duplicated the description of module names in the code now in the docstring of `find_matching_modules` (so the description is once in the code, once in the docs).